daemon/algod/api/server/v2/handlers.go

```diff
@@ -715,7 +715,7 @@ func (v2 *Handlers) GetAssetByID(ctx echo.Context, assetID uint64) error {
 	}
 
 	lastRound := ledger.Latest()
-	record, err := ledger.Lookup(lastRound, creator)
+	record, _, err := ledger.LookupWithoutRewards(lastRound, creator)
```

Comment:
It's ok to call the `ledger.LookupWithoutRewards`, but then you'll need to add the rewards.